.github/workflows/unit.yml

```diff
@@ -25,6 +25,13 @@ jobs:
           github.com:443
           objects.githubusercontent.com:443
           proxy.golang.org:443
+          cloud.google.com:443
+          golang.org:443
```

Comment:
Lets use *.golang.org:443 instead of the three different entries on the list.

Reply:
no longer needed.